reference/clinic.v1.yaml

```diff
@@ -1486,6 +1612,82 @@ components:
       $ref: ./clinic/models/membershiprestriction.v1.yaml
     MembershipRestrictions:
       $ref: ./clinic/models/membershiprestrictions.v1.yaml
+    EHRSettings:
+      $ref: ./clinic/models/ehrsettings.v1.yaml
+    EHRFacility:
+      $ref: ./clinic/models/ehrfacility.v1.yaml
+    EHRMatchRequest:
+      title: EHR Match Request
+      x-stoplight:
+        id: nq0mlm5dqdfw0
+      type: object
+      properties:
+        clinic:
+          $ref: '#/components/schemas/EHRMatchClinicRequest'
+        patient:
+          $ref: '#/components/schemas/EHRMatchPatientRequest'
+      required:
+        - clinic
+      x-examples:
```

Comment:
Nit: the samples would be easier to have in each model so every reference gets them